ext/oauth-client/info.xml

```diff
@@ -15,7 +15,7 @@
     <url desc="Licensing">http://www.gnu.org/licenses/agpl-3.0.html</url>
   </urls>
   <releaseDate>2020-10-23</releaseDate>
-  <version>5.39.alpha1</version>
+  <version>5.39.alpha2</version>
```

Comment:
Following up on @demeritcowboy's comment about versions -- This may be the rare case where someone had foresight to bump the`<version>` properly on a core-ext (nice @highfalutin) ... *except*... this *just* changed a couple days ago for 5.39+. Since it's easy to forget to increment this, we merged #20303 which will sync the `<version>` automatically. Now, when patching a core ext, you shouldn't explicitly update `<version>`. (It'll be updated in tandem with alpha/beta/stable milestones.)

Reply:
Fixed. Out of curiosity, how does the automatic version number relate to `<compatibility>`?